docs/source/user/parallel.rst

```diff
@@ -84,7 +84,11 @@ function/operator using its previous value in the loop body. The initial value
 of the reduction is inferred automatically for ``+=`` and ``*=`` operators.
 For other functions/operators, the reduction variable should hold the identity
 value right before entering the ``prange`` loop.  Reductions in this manner
-are supported for scalars and for arrays of arbitrary dimensions.
+are supported for scalars and for arrays of arbitrary dimensions. Care should be
+taken, however, as reducing into slices or elements of an array is not supported 
+if the elements specified by the slice or index is written to simultaneously by 
+multiple parallel threads. The compiler will not detect such cases and may result 
```

Comment:
The compiler does have some code for race detection so I might reword this as "The compiler may not detect such cases  and then a race condition would occur."